subprojects/docs/src/docs/userguide/artifactDependenciesTutorial.xml

```diff
@@ -157,7 +157,11 @@
         <sample id="defineMavenCentral" dir="userguide/artifacts/defineRepository" title="Usage of Maven central repository">
             <sourcefile file="build.gradle" snippet="maven-central"/>
         </sample>
-        <para>Or a remote Maven repository:</para>
+        <para>Or Bintray's JCenter:</para>
+        <sample id="defineJCenter" dir="userguide/artifacts/defineRepository" title="Usage of Jcenter repository">
```

Comment:
Pick J**C**enter or J**c**enter and use the same capitalization on both lines 160 and 161.